tools/preview/preview.js

```diff
@@ -5,6 +5,14 @@ import {
   withReducedMotionWrapper
 } from './decorators/index.js';
 
+import '@storybook/addon-console';
+import { setConsoleOptions } from '@storybook/addon-console';
+
+const panelExclude = setConsoleOptions({}).panelExclude;
+setConsoleOptions({
+  panelExclude: [...panelExclude, /deprecated/, /TypeError/],
```

Comment:
if panelExclude is spreading objects then /deprecated/ and /TypeError/ will be invalid types?
Do you want to keep a type check?

Also if panelExclude is returning null or undefined this line can throw error: Cannot spread on null

Reply:
A note for later, I got the implementation from here: https://github.com/storybookjs/storybook-addon-console#storybookaddon-consolesetconsoleoptionsoptionsorfn--addonoptions. The authors were on the storybook team so I suspect panelExclude always returns a value. it's a great point though.  We could add: `const panelExclude = setConsoleOptions({}).panelExclude || [];` just to be safe.